tests/test_registry.py

```diff
@@ -0,0 +1,34 @@
+import os
+from urllib.parse import urlsplit
+
+from aiidalab.fetch import GitRepo, fetch_from_url
+from aiidalab.registry.releases import _get_release_commits, _split_release_line
+
+
+def test_get_all_tagged_releases():
+    """Test that all tagged releases are returned."""
+    url = "git+https://github.com/aiidalab/aiidalab-qe.git@*:v23.04.0^.."
+    base_url, release_line = _split_release_line(url)
+    parsed_url = urlsplit(base_url)
+
+    with fetch_from_url(base_url) as repo_path:
+        if parsed_url.scheme.startswith("git+"):
```

Comment:
Surely this if is not needed when we have the URL defined above?

Reply:
Yep! Removed.